src/CommandHandling/AuthorizedCommandBus.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace CultuurNet\UDB3\CommandHandling;
+
+use Broadway\CommandHandling\CommandBusInterface;
+use CultuurNet\UDB3\Offer\Commands\AuthorizableCommandInterface;
+use CultuurNet\UDB3\Security\CommandAuthorizationException;
+use CultuurNet\UDB3\Security\SecurityInterface;
+use CultuurNet\UDB3\Security\UserIdentificationInterface;
+
+class AuthorizedCommandBus extends CommandBusDecoratorBase implements AuthorizedCommandBusInterface
+{
+    /**
+     * @var UserIdentificationInterface
+     */
+    private $userIdentification;
+
+    /**
+     * @var SecurityInterface
+     */
+    private $security;
+
+    /**
+     * AuthorizedCommandBus constructor.
+     * @param CommandBusInterface $decoratee
+     * @param UserIdentificationInterface $userIdentification
+     * @param SecurityInterface $security
+     */
+    public function __construct(
+        CommandBusInterface $decoratee,
+        UserIdentificationInterface $userIdentification,
+        SecurityInterface $security
+    ) {
+        parent::__construct($decoratee);
+
+        $this->userIdentification = $userIdentification;
+        $this->security = $security;
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function dispatch($command)
+    {
+        $authorized = true;
```

Comment:
Inderdaad beter om enkel op true te zetten indien geen AuthorizableCommandInterface en dus de default value weg te halen. Is nu zo aangepast.